praetorian_cli/handlers/hunt.py

```
@@ -0,0 +1,477 @@
+    click.echo(f'  Created:        {h.get("created", "?")}')
+    click.echo(f'  Created By:     {h.get("createdBy", "?")}')
+    click.echo(f'  Expires:        {h.get("expiresAt", "?")}')
+    click.echo(f'  Agent:          {h.get("agent", "hannibal")}')
+    click.echo(f'  Iterations:     {h.get("iterationCount", 0)}')
+    click.echo(f'  Findings:       {h.get("findingsCount", 0)}')
+    click.echo(f'  Last Started:   {h.get("lastStartedAt", "—")}')
+    click.echo(f'  Last Completed: {h.get("lastCompletedAt", "—")}')
+
+    prompt = h.get('prompt', '')
+    if prompt:
+        click.echo(f'  Mandate:')
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)